admin/lib/tasks/tailwind.rake

```diff
@@ -0,0 +1,39 @@
+namespace :solidus_admin do
+  namespace :tailwindcss do
+    require "tailwindcss-rails"
+
+    def run(args = "")
```

Comment:
I don't remember how rake works with regards to methods, is this defined on top-level or inside the rake task?

Reply:
Great call! I was under the assumption that it created an anonymous class and bound to it, but it was polluting the `Object` namespace. I extracted to a helper module. Thanks for pointing out.